toolkit/tools/pkgworker/pkgworker.go

```diff
@@ -85,14 +89,36 @@ func main() {
 	defines[rpm.DistroReleaseVersionDefine] = *distroReleaseVersion
 	defines[rpm.DistroBuildNumberDefine] = *distroBuildNumber
 	defines[rpm.MarinerModuleLdflagsDefine] = "-Wl,-dT,%{_topdir}/BUILD/module_info.ld"
-	if *useCcache {
-		defines[rpm.MarinerCCacheDefine] = "true"
+
+	ccacheManager, err := ccachemanagerpkg.CreateManager(*ccacheRootDir, *ccachConfig)
+	if err == nil {
+		if *useCcache {
+			buildArch, err := rpm.GetRpmArch(runtime.GOARCH)
+			if err == nil {
+				err = ccacheManager.SetCurrentPkgGroup(*basePackageName, buildArch)
+				if err == nil {
+					if ccacheManager.CurrentPkgGroup.Enabled {
+						defines[rpm.MarinerCCacheDefine] = "true"
+					}
+				} else {
+					logger.Log.Warnf("Failed to set package ccache configuration:\n%v", err)
```

Comment:
use `%w` instead of `%v` in this file?

Reply:
I get an error `(*github.com/sirupsen/logrus.Logger).Warnf does not support error-wrapping directive %w`.